src/js/subtraction/components/Selector.js

```diff
@@ -1,27 +1,91 @@
-import { map } from "lodash-es";
 import PropTypes from "prop-types";
 import React from "react";
 import { MultiSelector, MultiSelectorItem } from "../../base/MultiSelector";
 import { Link } from "react-router-dom";
 import { NoneFoundBox } from "../../base/NoneFound";
+import styled from "styled-components";
+import { BoxGroupSection, Icon, Input } from "../../base";
 
-export const SampleSubtractionSelector = ({ name, noun, selected, subtractions, onChange }) => {
-    const subtractionComponents = map(subtractions, ({ name, id }) => (
-        <MultiSelectorItem key={id} name={name} value={id} id={id}>
-            <span>{name}</span>
-        </MultiSelectorItem>
-    ));
-    if (!subtractions.length) {
+export const SampleInputContainer = styled.div`
+    position: relative;
+    display: flex;
+`;
+
+export const IconContainer = styled.div`
+    position: absolute;
+    margin-left: 92%;
+    margin-top: 10px;
+`;
+
+export const WorkflowItem = styled(BoxGroupSection)`
+    display: flex;
+    flex-direction: row;
+    border: 1px solid;
+    border-color: ${props => props.theme.color.greyLight};
+`;
+
+export const SampleSubtractionSelector = ({
+    name,
+    noun,
+    selected,
+    subtractions,
+    onChange,
+    requireCheckbox,
+    requireInput,
+    term,
+    setTerm,
+    subtractionOptions
+}) => {
+    const subtractionComponents = subtractions.map(item => {
+        const result = item.id ? item : item.item;
         return (
-            <NoneFoundBox noun="subtractions">
-                <Link to="/subtractions"> Create one</Link>.
-            </NoneFoundBox>
+            <MultiSelectorItem
+                key={result.id}
+                name={result.name}
+                value={result.id}
+                id={result.id}
+                requireCheckbox={requireCheckbox}
+            >
+                <span>{result.name}</span>
+            </MultiSelectorItem>
         );
-    }
+    });
+
     return (
-        <MultiSelector name={name} noun={noun} selected={selected} onChange={onChange}>
-            {subtractionComponents}
-        </MultiSelector>
+        <>
+            {requireInput === true && (
```

Comment:
Comparison to true is probably redundant. Second statement will be included so long as `requireInput` is truthy.